libraries/Microsoft.Bot.Builder/TurnContext.cs

```diff
@@ -214,16 +208,14 @@ public bool Responded
         public Task<ResourceResponse[]> SendActivities(IActivity[] activities)
         {
             if (activities == null)
-            {
                 throw new ArgumentNullException(nameof(activities));
-            }
 
             if (activities.Length == 0)
             {
                 throw new ArgumentException("Expecting one or more activities, but the array was empty.", nameof(activities));
             }
 
-            var conversationReference = GetConversationReference(this.Activity);
+            var conversationReference = GetConversationReference(Activity);
```

Comment:
Passing the local property to the location fuction seems weird. 

Is GetconversationReference static?

Reply:
yes static so i think this is good (i actually like the static private helper pattern used here so I'm happy to keep that)